### ISSUE TITLE
Problems building on Ubuntu

### ISSUE THREAD

**Issue by jongeorge1999** [hyperlogic/splatapult#26]
![image](https://github.com/user-attachments/assets/2c9b72df-f2dd-4fec-b6ee-5d9bbd8ec78a)

Followed instructions in BUILD.md, can't seem to build it due to openxr.

**Comment by Bertani545** [comment]
Yeah, I had the same problem. It kinda "fixed" by  adding -DCMAKE_MAKE_PROGRAM=/usr/bin/make when executing. You also have to install libxcb-glx.
After that, I got an error where vcpkg/buildtrees/libxcrypt/config-x64-linux-dbg-config.log was not found.

**Comment by shadygm** [comment]
@jongeorge1999 and @Bertani545 could you check my most recent PR and let me know if the build issue has been fixed?
